src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -101,49 +92,32 @@ export const ConfirmSummary = ({
   const { priceImpactPercentage } = usePriceImpact(activeQuote)
   const walletSupportsBuyAssetChain = useWalletSupportsChain(buyAsset.chainId, wallet)
 
-  const userAddress = useMemo(() => {
-    if (!initialSellAssetAccountId) return ''
-
-    return fromAccountId(initialSellAssetAccountId).account
-  }, [initialSellAssetAccountId])
-
-  const { data: _isSmartContractSellAddress, isLoading: isSellAddressByteCodeLoading } =
```

Comment:
To confirm, we feel safe removing the sc sell address checks since the addition of the sc checks in account detection resulting in only 1 account enabled for sc wallets correct?

Was there a clear of the portfolio after the sc checks went in? I want to make sure there isn't a case where a user would have other evm accounts in their portfolio with an sc wallet.

I validated account management is disabled for wc as is switch chain. Feels safe, just want to gaugue your confidence level here. Thanks

Reply:
That's correct @kaladinlight! There was a migration in https://github.com/shapeshift/web/commit/fff7c3b7568#diff-d27dbffc4f1f65efb1979356cc8a0436c4b056c8ab4235820774898e33b60cf2R97 that has cleared wallet meta/data for wc wallets.

Feeling pretty confident here for this precise reason, as having a sc wallet means you won't have any account on any other chain (or worst case scenario if our endpoints are down, sc checks will succeed with llamarpc fallback, but unchained will fail meaning there will be no account at all).